sdk/servicebus/azure-servicebus/azure/servicebus/aio/management/_management_client_async.py

```diff
@@ -824,12 +829,14 @@ def list_rules(
         except AttributeError:
             subscription_name = subscription
 
-        def entry_to_rule(entry):
+        def entry_to_rule(ele, entry):
```

Comment:
why aren't we using this helper above?  it looks almost identical to the places you inserted deserialize_rule_key_values.

I'd also ask why this isn't grouped with the other helpers (either them here or this there; especially since this doesn't require "self" the latter seems more natural?)

Reply:
Yes it can be reused. The entry_to_xxx is basically a one-line statement (or can be changed to one line) so I didn't extract them into helper functions.
The entry_to_rule now becomes two lines. But it will become one line again once the autorest extension is released.

Reply:
To be precise, I meant if like, around line 738, could use this function as well.  But if that would be making work four ourselves in the future (If I'm understanding your comment right) then we can not bikeshed on this for now.